pkg/keccak/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME    := keccak
+PKG_URL     := https://github.com/gvanas/KeccakCodePackage
+PKG_VERSION := bc592d1780a6b3204c26c6f872552ace8fc6fd65
+
+ifeq ($(CPU),native)
+  CPU_ARCHIVE_NAME := Native
+else
+ifeq ($(CPU_ARCH),$(filter $(CPU_ARCH), cortex-m0 cortex-m0plus))
+  CPU_ARCHIVE_NAME := ARMv6M
+else
+ifeq ($(CPU_ARCH),$(filter $(CPU_ARCH), cortex-m3 cortex-m4 cortex-m4f cortex-m7 cortex-m7f))
```

Comment:
```suggestion
ifneq (,$(filter $(CPU_ARCH), cortex-m3 cortex-m4 cortex-m4f cortex-m7 cortex-m7f))
```

Reply:
Well, I was considering just querying `CPU_ARCH` straight away instead of `CPU_CORE`.